tests/Tests/Scrapers/BullereiTest.php

```diff
@@ -0,0 +1,36 @@
+<?php
+namespace JK\Mittagstisch\Tests;
+
+use JK\Mittagstisch\Scrapers\Bullerei;
+
+class BullereiTest extends \PHPUnit_Framework_TestCase
+{
+    /** @var Bullerei */
+    private $instance;
+
+    public function setUp()
+    {
+        $this->instance = new Bullerei();
+    }
+
+    public function testInstance()
+    {
+        static::assertInstanceOf('JK\Mittagstisch\Scrapers\Bullerei', $this->instance);
+    }
+
+    public function testScrape()
+    {
+        $menus = $this->invokeScrape($this->instance, 'scrape');
+        static::assertInternalType('array', $menus);
+        static::assertContainsOnlyInstancesOf('JK\Mittagstisch\MenuItem', $menus);
```

Comment:
Technically this is a functional test and not a unit test anymore. If that's the case, the network code should be stubbed.